streamly-process.cabal

```diff
@@ -82,7 +82,7 @@ benchmark Benchmark.System.Process
     , directory         >= 1.2.2 && < 1.4
     , gauge             >= 0.2.4 && < 0.3
     , process           >= 1.0   && < 1.7
-    , streamly          >= 0.7.1 && < 0.8
+    , streamly          >= 0.8   && < 0.9
```

Comment:
We are depending on the internal imports. Do you want to make the upper bound inclusive of the latest working minor release? `0.8.0` in this case.